src/compiler/scala/tools/nsc/symtab/SymbolLoaders.scala

```diff
@@ -281,6 +281,8 @@ abstract class SymbolLoaders {
     }
   }
 
+  val completedClassfiles = scala.collection.mutable.ListBuffer[AbstractFile]()
```

Comment:
This should be cleared at the end of a `Run`, either by storing it as a field of `Run`, or by registering it with `perRunCaches`.